cdbot/cogs/fun.py

```diff
@@ -127,29 +140,23 @@ def check(reaction, user):
 
         # React if a message contains an @here or @everyone mention.
         if any(mention in message.content for mention in ("@here", "@everyone")):
-            await message.add_reaction("🙁")
+            await message.add_reaction("\N{SLIGHTLY FROWNING FACE}")
             await emojify(message, "who pinged")
 
-        # React if message contains dabato.
-        if "dabato" in message.content:
-            await message.add_reaction("🤔")
-
         # React FBI OPEN UP if message contains trigger words.
         triggers = ["child", "fbi", "loli", "hentai", "illegal", "maltego"]
         if any(trigger in message.content.lower() for trigger in triggers):
             await emojify(message, "fbi open up")
 
-        # React if message contains Kali.
-        if "kali" in message.content.lower():
-            await message.add_reaction("🚔")
-
-        # React if message contains Duck.
-        if "duck" in message.content.lower():
-            await message.add_reaction("🦆")
+        for trigger in REACT_TRIGGERS:
+            # Check if the message contains a trigger
+            if trigger in message.content.lower():
```

Comment:
Would lowercasing outside of the loop be more efficient as we'd only need to do it once?